code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1170,6 +1170,25 @@ Traitors and the like can also be revived with the previous role mostly intact.
 				target.ghostize(FALSE,SENTIENCE_FORCE)
 			else
 				target.set_playable()
+		if(ADMIN_PUNISHMENT_TOE)
+			if(!ishuman(target))
+				to_chat(usr, "<span class='warning'>Only humanoids can stub their toes!</span>")
+				return
+			var/mob/living/carbon/human/H = target
+			to_chat(H, "<span class='warning'>You stub your toe on an invisible table!</span>")
```

Comment:
You should move toe stubbing into its own proc rather than copypasting the code here.